resident-services/scripts/Resident_Test_script.jmx

```diff
@@ -3619,6 +3619,870 @@ vars.put(&quot;utcTime&quot;, utcTime);</stringProp>
           <hashTree/>
         </hashTree>
       </hashTree>
+      <ThreadGroup guiclass="ThreadGroupGui" testclass="ThreadGroup" testname="Login (Execution)" enabled="true">
```

Comment:
Login (Execution) - > we should send the code here, it should have a prepartion using the esignet APIs

Reply:
We are not using Esignet API preparation here. Because we don't need to send code here.
This Login API is used to redirect to Esignet UI.

Reply:
URL's for both the Login are different.

Reply:
this is fine, lets check the login URL correctness with v2 and it has to be asserted with 302 response code

Reply:
Checked with swagger the correct URL is - https://api-internal.cellbox.mosip.net/resident/v1/login/ only. There is no v2 part added in the URL.
PFA 
![image](https://github.com/mosip/performance-test-scripts/assets/107672790/d766c553-4f22-4e92-916e-d670403bf3c9)


And the response assertion is added in the same way as Login Redirect URI API with only title name is e-signet for the above Login API.

Reply:
Also it gives back a 302 

![image](https://github.com/mosip/performance-test-scripts/assets/9315119/62858721-7c56-4e80-a67d-bca81de0270b)

Reply:
Here also will get 2 samples in response where 1 is 302 and another is 200.
PFA 
![image](https://github.com/mosip/performance-test-scripts/assets/107672790/6686fc97-d6fc-46df-bd7a-d45836de6dd0)
![image](https://github.com/mosip/performance-test-scripts/assets/107672790/ef73e8c3-11ce-4d79-81c4-0927af9169a4)

Reply:
then response time is addition of both response times ? If yes then that is not correct. We have to make call only to this API and check for the 302 reponse and record response time of only this API endpoint

Reply:
ok, so v1 is fine, but we need the response time only for the login API with response code check for 302

Reply:
Have updated the response assertion with response code check for 302 only. So for Login API there is only one assertion check.